guide/installation-and-configuration/upgrading-to-37.markdown

```diff
@@ -120,21 +196,21 @@ If everything looks good, you are ready to upgrade the clients, please skip to P
     ```
 3. Install the new CFEngine Policy Server package (you may need to adjust the package name based on CFEngine edition, version and distribution).
   * ```console
-    rpm -i cfengine-nova-hub-3.6.1.x86_64.rpm # Red Hat based distribution
+    rpm -i cfengine-nova-hub-3.7.0.x86_64.rpm # Red Hat based distribution
     ```
   * ```console
-    dpkg --install cfengine-nova-hub_3.6.1-1_amd64.deb # Debian based distribution
+    dpkg --install cfengine-nova-hub_3.7.0-1_amd64.deb # Debian based distribution
     ``` 
 4. Restore ppkeys and any Enterprise license file (default location is /var/cfengine/masterfiles/license.dat) from backup.
```

Comment:
I dont think we have to restore the keys, we arent having them uninstall the hub package anymore right?